courses/factories.py

```diff
@@ -1,13 +1,18 @@
-# pylint: disable=missing-docstring
+"""Factories for testing"""
 import factory
 from factory.django import DjangoModelFactory
 from .models import Module, Course
 
+
 class CourseFactory(DjangoModelFactory):
-    class Meta:
+    """Factory for Course"""
```

Comment:
Broader question about docstrings.. Does this actually provide value? Is anyone ever going to look at a class called `CourseFactory`, not know what it does and be satisfied with the explanation of "Factory for Course"?